mk/variables.mk

```diff
@@ -55,9 +55,9 @@ export MV     := mv -f
 export PRINTF := printf
 export MKDIR  := mkdir -p
 export LN     := ln -s
-export MD5    := $(shell for i in md5 md5sum; do type $$i >/dev/null 2>&1 && echo $$i && break; done)
-export CPIO   := $(shell for i in gcpio cpio; do type $$i >/dev/null 2>&1 && echo $$i && break; done)
-export AWK    := $(shell for i in gawk awk nawk mawk; do type $$i >/dev/null 2>&1 && echo $$i && break; done)
+export MD5    := $(shell for i in md5 md5sum; do type $$i >/dev/null 2>&1 && echo $$i && exit; done; echo md5sum)
```

Comment:
May be add warning about missing utillity and taking default one?
Refactor check to make function?
I made these changes in #924, please, check